scripts/generator/graphql_generator/csharp/SDK/Serializable.cs.erb

```diff
@@ -4,10 +4,11 @@ namespace <%= namespace %>.SDK {
     using System.Collections;
     using System.Collections.Generic;
     using UnityEngine;
+    using <%= namespace %>.MiniJSON;
 
     public class Serializable {
         public string ToJsonString() {
-            return JsonUtility.ToJson(this);
+            return Json.Serialize(this);
```

Comment:
`MiniJSON.Serialize` expects the passed in param to implement `IDictionary` or `IList`.